### PR TITLE
UI changes

### PR DESCRIPTION
**Pull request by Capn05**
Added navigation link to all pages on website header
Added Profile page for user inputted metadata and and movie-logging flow page to start technical challenge #1
small UI changes throughout all pages

### CODE REVIEW

**Comment by samyalihamad** [comment]
UI is looking good. A few high level reminders on PRs are to add good descriptions to help the reviewer understand if the changes match what you expect the code to be doing. For example, this latest PR says you started on the Profile page but it doesn't mention what the designs are for that page and what you completed. If its already mentioned in the project doc you can link the section to it. Having good descriptions and screenshots will also help with telling the story of your progression. I would also make the titles descriptive so it is easy to know what each PR contains. For example, this PR says UI Changes but we can't tell this is the PR used to add the Profile page.

File: src/FlixterHeader.css [hunk 1]
```diff
@@ -15,26 +15,38 @@
   height: 100px;
 }
 
-.title {
+/* .title {
```
Comment (samyalihamad) [inline src/FlixterHeader.css:18]: Be sure to not commit commented out code.

File: src/MovieLogger.jsx [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+  const handleRatingSelect = (e) => {
+    setCurrentMovie({ ...currentMovie, rating: e.target.value });
+  };
+
+  const startComparison = () => {
+    if (currentMovie.name && currentMovie.rating) {
+      setIsComparing(true);
+      setComparisonIndex(0);
+    } else {
+      alert("Please enter movie details and select a rating.");
```
Comment (samyalihamad) [inline src/MovieLogger.jsx:23]: As an addition, consider using one of the free react notification libraries

File: src/FlixterHeader.css (hunk 1 above)
Comment (BriptimusPrimus) [reply src/FlixterHeader.css:18]: +1

File: src/App.jsx [hunk 3]
```diff
@@ -60,7 +61,22 @@ function App() {
...
+                <ProtectedRoute>
+                  <Profile></Profile>
+                </ProtectedRoute>
+              }
+            />
+            <Route
+              path="/MovieLogger"
+              element={
+                <ProtectedRoute>
+                  <MovieLogger></MovieLogger>
```
Comment (BriptimusPrimus) [inline src/App.jsx:76]: `<MovieLogger / >`

File: src/FlixterHeader.jsx [hunk 4]
```diff
@@ -15,17 +15,24 @@ function FlixterHeader({ likedMovies, watchedMovies }) {
       <NavLink className="title" to={"/home"}>
         <img className="logo" src={logo} alt="" />
       </NavLink>
+      <div className="links">
+        <NavLink className="navLink" to={"/myRankings"}>
+          Watched Movies
+        </NavLink>
+        {/* <NavLink className="navLink" to={"/recommendations"}>
```
Comment (BriptimusPrimus) [inline src/FlixterHeader.jsx:22]: Be sure to not commit commented out code.

File: src/MovieLogger.jsx (hunk 2 above)
Comment (BriptimusPrimus) [reply src/MovieLogger.jsx:23]: @samyalihamad I think this can be planned as a stretch goal for the project.

File: src/MovieLogger.jsx [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+    setComparisonIndex(0);
+  };
+
+  return (
+    <div>
+      <FlixterHeader />
+      <div className="loggingContainer">
+        <h1>Movie Logger</h1>
+        {!isComparing ? (
+          <div>
```
Comment (BriptimusPrimus) [inline src/MovieLogger.jsx:57]: Can you try replacing this div with a fragment? (`<>`)
The closing div at line 72 should be replaced with `</>`

Does it look the same, or is the div responsible for styling the UI the way you need?

**Comment by BriptimusPrimus** [comment]
It would be good to see screenshots of the new UI changes in the PR description, and to adopt this practice in all PRs

File: src/FlixterHeader.css (hunk 1 above)
Comment (Capn05) [reply src/FlixterHeader.css:18]: Ok got it! bad habit

File: src/MovieLogger.jsx (hunk 2 above)
Comment (Capn05) [reply src/MovieLogger.jsx:23]: Sounds good I can add it to the project plan

**Pull request merged**